packages/components/psammead-useful-links/src/index.test.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { shouldMatchSnapshot } from '@bbc/psammead-test-helpers';
+import { latin } from '@bbc/gel-foundations/scripts';
+import { UsefulLink, UsefulLinksLi, UsefulLinksUl } from './index';
+
+const usefulCaptions = [
+  {
+    name: 'Mitocinmu da sauko da sautin labarai',
+    url: 'https://www.bbc.com/igbo/afirika-49883577',
+  },
+  {
+    name: 'Labaran BBC Hausa a text',
+    url: 'https://www.bbc.com/igbo/afirika-49872694',
+  },
+  {
+    name: 'Abokan huldar BBC Hausa',
+    url: 'https://www.bbc.com/igbo/afirika-49869003',
+  },
+  {
+    name: 'Timi Frank: Osinbajo ya maka mutum biyu',
+    url: 'https://www.bbc.com/igbo/afirika-49883189',
+  },
+  {
+    name: 'Gwaninta ba ta karbi wani dan Nijeriya',
+    url: 'https://www.bbc.com/igbo/afirika-49869001',
+  },
+];
+
+describe('One useful link', () => {
+  shouldMatchSnapshot(
+    'should render correctly',
+    <UsefulLink script={latin} service="news" href={usefulCaptions[0].url}>
+      {usefulCaptions[0].name}
+    </UsefulLink>,
+  );
+});
+
+describe('Multiple useful links', () => {
+  shouldMatchSnapshot(
+    'should render correctly',
+    <UsefulLinksUl>
+      {usefulCaptions.map(item => {
+        return (
+          <UsefulLinksLi key={usefulCaptions.indexOf(item)}>
```

Comment:
The function passed to `map` receives `index` as the 2nd argument

Line 42: `usefulCaptions.map((item, index) => {`

Line 44: `<UsefulLinksLi key={index}>`

Reply:
@ryanmccombe  Using `index` would cause violation of `react/no-array-index-key`. `usefulCaptions.indexOf(item)` has been used as a workaround for that.

Reply:
It does, I just didn't add an `index` to `map` here because we definitely won't do it when we use `UsefulLinks` component. When we filter through data, there will be ids that we can pass to the `key` attribute. So I am not sure if it's applicable adding an `index` here

Reply:
👍 fair enough, it was a minor point - but we are using an index here - we're just going through the extra `indexOf` step to get it :)